### PR TITLE
[release-8.1] HTML fixes

### PR DESCRIPTION
**Pull request by EreMaijala**
This is a set of commits that fix various HTML validation issues.

### CODE REVIEW

**Comment by EreMaijala** [comment]
Mink tests are passing for me.

File: themes/bootstrap3/templates/record/cart-buttons.phtml [hunk 1]
```diff
@@ -9,14 +9,15 @@
...
     <noscript>
-      <form method="post" name="addForm" action="<?=$this->url('cart-processor')?>">
-        <input type="hidden" name="ids[]" value="<?=$this->escapeHtmlAttr($cartId)?>" />
-        <?php if ($cart->contains($cartId)): ?>
-          <input class="btn btn-default" type="submit" name="delete" value="<?=$this->transEscAttr('Remove from Book Bag')?>"/>
-        <?php else: ?>
-          <input class="btn btn-default" type="submit" name="add" value="<?=$this->transEscAttr('Add to Book Bag')?>"/>
-        <?php endif; ?>
-      </form>
+      <?php if (!$cart->contains($cartId)): ?>
```
Comment (demiankatz) [inline themes/bootstrap3/templates/record/cart-buttons.phtml:12]: Exactly what problem is being solved here? I'm not too keen on GET links with side effects -- maybe it's better to just leave this alone for now and then remove the `<noscript>` entirely in VuFind 9.

**Comment by demiankatz** [comment]
@EreMaijala, this all looks good to me apart from [a851d80](https://github.com/vufind-org/vufind/pull/2476/commits/a851d80b56b6ce351d6706eef0d5d3190bfc6cc3), which I think might be a bit questionable. Unless you feel otherwise, perhaps I should cherry-pick all the other commits to release-8.1, and we can figure out what to do about no-JS cart buttons in a separate PR targeted at dev.

File: themes/bootstrap3/templates/record/cart-buttons.phtml (hunk 1 above)
Comment (EreMaijala) [reply themes/bootstrap3/templates/record/cart-buttons.phtml:12]: The problem is that you can't have a form inside noscript. Also, the buttons look seriously bad among the other links. I'll be happy to drop this one if you think it's better to keep the form thing.

**Comment by EreMaijala** [comment]
@demiankatz Works for me. It'll be small change to get rid of the noscript part in dev.

**Comment by demiankatz** [comment]
As discussed, all relevant commits have been cherry-picked to release-8.1, and the remaining functionality is superseded by #2477. Thanks again, @EreMaijala!

**Pull request closed without merging**